tests/unit/s2n_security_policies_test.c

```diff
@@ -59,9 +62,130 @@ int main(int argc, char **argv)
             }
         }
 
-        /* If a TLS 1.3 Cipher is present in the Security Policy, then the minimum required TLS 1.3 signature algorithms
-         * must be present as well. */
         if (has_tls_13_cipher) {
+            /* Validate that s2n_tls13_default_sig_scheme() is successful on all TLS 1.3 Security Policies for all
+             * TLS 1.3 Ciphers */
+            {
+                struct s2n_cipher_suite tls_13_ciphers[] = { s2n_tls13_aes_128_gcm_sha256,
+                                                             s2n_tls13_aes_256_gcm_sha384,
+                                                             s2n_tls13_chacha20_poly1305_sha256 };
+
+                for (int i = 0; i < s2n_array_len(tls_13_ciphers); i ++) {
+                    char *cert_chain;
+                    char *private_key;
+                    struct s2n_cert_chain_and_key *default_cert;
+
+                    EXPECT_NOT_NULL(cert_chain = malloc(S2N_MAX_TEST_PEM_SIZE));
+                    EXPECT_NOT_NULL(private_key = malloc(S2N_MAX_TEST_PEM_SIZE));
+                    EXPECT_SUCCESS(s2n_read_test_pem(S2N_DEFAULT_TEST_CERT_CHAIN, cert_chain, S2N_MAX_TEST_PEM_SIZE));
+                    EXPECT_SUCCESS(s2n_read_test_pem(S2N_DEFAULT_TEST_PRIVATE_KEY, private_key, S2N_MAX_TEST_PEM_SIZE));
+                    EXPECT_NOT_NULL(default_cert = s2n_cert_chain_and_key_new());
+                    EXPECT_SUCCESS(s2n_cert_chain_and_key_load_pem(default_cert, cert_chain, private_key));
```

Comment:
Try this instead: https://github.com/aws/s2n-tls/blob/main/tests/testlib/s2n_testlib.h#L161-L162